controllers/comments.controller.js

```diff
@@ -2,13 +2,14 @@ const {returnAllCommentsByArticleId, addNewCommentByArticleIdToDatabase, updateC
 
 exports.getAllCommentsByArticleId = (req, res, next) => {
     const {article_id} = req.params
-    returnAllCommentsByArticleId(article_id).then((articles) => {
-        res.status(200).send(articles)
+    returnAllCommentsByArticleId(article_id).then((comments) => {
+        res.status(200).send(comments)
```

Comment:
Make sure you are retuning your data on an appropriate key e.g. {comments: comments}. You may need to tweak your tests to reflect this too.